opencog/atomese/interpreter/Interpreter.cc

```diff
@@ -139,7 +140,52 @@ ValuePtr Interpreter::execute(const Type t, const ValueSeq& params)
 		result = logical_not( LinkValueCast(params[0]));
 		return ValuePtr(result);
 	}
-	return ValuePtr();
+
+	if (t == COND_LINK) {
+        ValueSeq result, conds, exps, default_exp, res2;
+        std::vector<double> result2, res;
+        if (params.size() == 0)
+            throw SyntaxException(TRACE_INFO,
+                                  "CondLink is expected to be arity greater-than 0!");
+
+        if (params.size() == 1) {
+            default_exp.push_back(params[0]);
+        }
+
+        // If the conditions and expressions are flattened in even and odd
+        // positions respectively.
+        for(unsigned i = 0; i < params.size(); ++i) {
+
+            if(i % 2 == 0) {
+                if(i == params.size() - 1){
```

Comment:
Missing space between `)` and `{`, see https://wiki.opencog.org/w/Development_standards#Formatting